.travis.yml

```diff
@@ -3,23 +3,17 @@ os:
   - linux
   - osx
 julia:
-  - 0.7
```

Comment:
Removing testing for 0.7 requires that we remove support for 0.7. If I understand the automatic merge guidelines, that constitutes a breaking release so we'd need to go to 2.0.0. But I'm not certain.

Reply:
Ok, I didn't mean to be breaking. I thought 0.7 would be ok to remove given it was so temporary. But I like robust logic, so I'll add it back for now